core-bundle/src/Resources/contao/templates/analytics/analytics_matomo.html5

```diff
@@ -0,0 +1,23 @@
+<?php
+
+// To use this script, please fill in your Matomo site ID and Matomo URL below
+$MatomoSite = 0;
+$MatomoPath = '//www.example.com/matomo/';
+
+// DO NOT EDIT ANYTHING BELOW THIS LINE UNLESS YOU KNOW WHAT YOU ARE DOING!
+if ($MatomoSite > 0 && '//www.example.com/matomo/' != $MatomoPath && !BE_USER_LOGGED_IN && !$this->hasAuthenticatedBackendUser()): ?>
+
+<script>
+  var _paq = window._paq || [];
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u='<?= $MatomoPath ?>';
+    _paq.push(['setTrackerUrl', u+'matomo.php']);
+    _paq.push(['setSiteId', <?= $MatomoSite ?>]);
+    var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+    g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'matomo.js'; s.parentNode.insertBefore(g,s);
```

Comment:
```suggestion
    g.async=true; g.defer=true; g.src=u+'matomo.js'; s.parentNode.insertBefore(g,s);
```

Since Contao always generates HTML5 pages, the type attribute can be omitted.